site/_resources/upgrading.md

```diff
@@ -18,8 +18,7 @@ The <code>IngressRoute</code> CRD has been deprecated and will not receive furth
 <p>
 </p>
 Please see the documentation for <a href="/docs/v1.0.0/httpproxy"><code>HTTPProxy</code></a>, which is the successor to <code>IngressRoute</code>.
-You can also read the <a href="{% link _guides/ingressroute-to-httpproxy.md %}">IngressRoute to HTTPProxy upgrade</a> guide.
-</p>
+You can also read the <a href="/guides/ingressroute-to-httpproxy">IngressRoute to HTTPProxy upgrade</a> guide.
```

Comment:
Can we keep this `{% link %}`?

Reply:
I missed that changed item. I'll set as {% link %} again